lib/server/generate.js

```diff
@@ -247,11 +247,9 @@ async function execute() {
       !commander.skipImageCompression
     ) {
       const parts = normalizedFile.split(`${sep}static${sep}`);
-      const targetDirectory = join(
-        buildDir,
-        parts[1].substring(0, parts[1].lastIndexOf(sep))
```

Comment:
Was this always wrong? I see you removed this from your update.

Reply:
It's not really wrong but this is not declarative enough. I think using function from nodejs path clearly shows the intent. 

```js
const targetDirectory = path.dirname(targetFile);
```

The previous code in the below section
```js
mkdirp.sync(path.dirname(targetDirectory));
```

is wrong though. It should be
```js
mkdirp.sync(targetDirectory);
```